modular_skyrat/code/game/objects/items/tools/modular_tools.dm

```diff
@@ -0,0 +1,140 @@
+
+/obj/item/wirecutters/power
+	icon = 'modular_skyrat/icons/obj/tools.dmi'
+	lefthand_file = 'modular_skyrat/icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'modular_skyrat/icons/mob/inhands/equipment/tools_righthand.dmi'
+/obj/item/wirecutters/power/get_belt_overlay()
+	return mutable_appearance('modular_skyrat/icons/obj/clothing/belt_overlays.dmi', "jaws_cutter")
+
+// Electric Cutter
+
+/obj/item/crowbar/electric
+	name = "power cutter"
+	desc = "A compact electric prying and cutting tool. It's fitted with a prying head."
+	icon = 'modular_skyrat/icons/obj/tools.dmi'
+	icon_state = "pry"
+	item_state = "old_jawsoflife"
+	lefthand_file = 'modular_skyrat/icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'modular_skyrat/icons/mob/inhands/equipment/tools_righthand.dmi'
+	custom_materials = list(/datum/material/iron=100,/datum/material/titanium=5)
+/obj/item/crowbar/power/get_belt_overlay()
+	return mutable_appearance('modular_skyrat/icons/obj/clothing/belt_overlays.dmi', "old_jaws_pry")
+
+/obj/item/wirecutters/electric
+	name = "power cutter"
+	desc = "A compact electric prying and cutting tool. It's fitted with a cutting head."
+	icon = 'modular_skyrat/icons/obj/tools.dmi'
+	icon_state = "cutters"
+	item_state = "old_jawsoflife"
+	lefthand_file = 'modular_skyrat/icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'modular_skyrat/icons/mob/inhands/equipment/tools_righthand.dmi'
+	random_color = FALSE
+	custom_materials = list(/datum/material/iron=100,/datum/material/titanium=5)
+/obj/item/wirecutters/power/get_belt_overlay()
+	return mutable_appearance('modular_skyrat/icons/obj/clothing/belt_overlays.dmi', "old_jaws_cutter")
+
+///////////////// This is less intrusive than changing the try to pry proc in airlocks.dm /////////////////
+/obj/item/crowbar/electric/attack_self(mob/user)
+	playsound(get_turf(user), 'sound/items/change_jaws.ogg', 50, 1)
+	var/obj/item/wirecutters/electric/cutjaws = new /obj/item/wirecutters/electric(drop_location())
+	cutjaws.name = name // Skyrat fix
+	to_chat(user, "<span class='notice'>You attach the cutting jaws to [src].</span>")
+	qdel(src)
+	user.put_in_active_hand(cutjaws)
+/obj/item/wirecutters/electric/attack_self(mob/user)
+	playsound(get_turf(user), 'sound/items/change_jaws.ogg', 50, 1)
+	var/obj/item/crowbar/electric/pryjaws = new /obj/item/crowbar/electric(drop_location())
+	pryjaws.name = name // Skyrat fix
+	to_chat(user, "<span class='notice'>You attach the pry jaws to [src].</span>")
+	qdel(src)
+	user.put_in_active_hand(pryjaws)
+/obj/item/screwdriver/power/green/attack_self(mob/user)
+	playsound(get_turf(user),'sound/items/change_drill.ogg',50,1)
+	var/obj/item/wrench/power/b_drill = new /obj/item/wrench/power/green(drop_location())
+	to_chat(user, "<span class='notice'>You attach the bolt driver bit to [src].</span>")
+	qdel(src)
+	user.put_in_active_hand(b_drill)
+/obj/item/wrench/power/green/attack_self(mob/user)
```

Comment:
Please consider putting an empty line between separate procs, it makes stuff so much easier to read, instead of having a sudden big uninterrupted blob of code.

Reply:
done